sql/updates/auth/2015_09_23_00_auth.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `account` DROP COLUMN `mutetime`, DROP COLUMN `mutereason`, DROP COLUMN `muteby`; 
```

Comment:
sql/updates/auth/2015_09_23_00_auth.sql

I sugest you to name filenames as the PR id you are ussing ie:

sql/updates/auth/2015_09_23_15584_auth.sql